docs/src/api/class-touchscreen.md

```diff
@@ -20,3 +20,23 @@ Dispatches a `touchstart` and `touchend` event with a single touch at the positi
 ### param: Touchscreen.tap.y
 * since: v1.8
 - `y` <[float]>
+
+## async method: Touchscreen.touch
+* since: v1.46
+
+Synthesizes a touch event.
+
+### param: Touchscreen.touch.type
+* since: v1.46
+- `type` <[TouchType]<"touchstart"|"touchend"|"touchmove"|"touchcancel">>
+
+Type of the touch event.
+
+### param: Touchscreen.touch.touchPoints
```

Comment:
Since you advertise it as event generator, align with https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/touches.

```suggestion
### param: Touchscreen.touch.touches
```